src/main/db/queryHandler.js

```diff
@@ -0,0 +1,18 @@
+import { db } from '../db/databaseStart.js';
+const collection = 'MSM-Scenario';
+
+export class QueryHandler {
```

Comment:
I think maybe the database can use the singleton pattern intend of static methods. That might be a refactor for next iterations

Reply:
Later you can explain me how it would work